app/models/post.rb

```diff
@@ -281,6 +281,18 @@ def valid_board_section
     errors.add(:section, "must be in the post's board")
   end
 
+  def valid_coauthors
```

Comment:
You're not actually guaranteed that the person editing is post.user unfortunately, this might need to happen controller-side

Reply:
I think what I did was check against the post.user and joined_coauthors